src/main/resources/services/sitecontent/generate-response.ts

```diff
@@ -164,6 +164,10 @@ export const generateSitecontentResponse = ({
 
     const { content, locale } = target;
 
+    if (legacyContentToRedirect.includes(content.type)) {
+        return resolveLegacyContentRedirects(content);
+    }
```

Comment:
Lurer på om sjekken på innholdstype kan bakes inn i resolveLegacyContentRedirects-funksjonen, litt samme patterns som de andre funksjonene her for diverse edge-cases.

Reply:
Ja, good point. Rettet!